<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ValueDisplayBar</class>
 <widget class="QFrame" name="ValueDisplayBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="display1">
     <item>
      <widget class="QLabel" name="displayLabel1">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="displayNumber1"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="display2">
     <item>
      <widget class="QLabel" name="displayLabel2">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="displayNumber2"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="display3">
     <item>
      <widget class="QLabel" name="displayLabel3">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="displayNumber3">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="display4">
     <item>
      <widget class="QLabel" name="displayLabel4">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="displayNumber4"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="display5">
     <item>
      <widget class="QLabel" name="displayLabel5">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="displayNumber5"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="display6">
     <item>
      <widget class="QLabel" name="displayLabel6">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="displayNumber6"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="display7">
     <item>
      <widget class="QLabel" name="displayLabel7">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="displayNumber7"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="timeLabel">
       <property name="text">
        <string>Time Elapsed:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="timeNumber"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
